server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1889,6 +1889,8 @@ public void checkIfMinorCompactionNeededForLogs(List<DfsLogger> closedLogs) {
     }
   }
 
+  @SuppressFBWarnings(value = "UL_UNRELEASED_LOCK",
+      justification = "lock is released by caller calling finishClearingUnusedLogs method")
```

Comment:
If the caller is responsible for releasing it, then shouldn't the caller catch the RTE and release it in the calling code?

Reply:
I can see what that looks like

Reply:
@ctubbsii  I pushed an update in 851271a based on your suggestion. I think that looks good.